.vitepress/data/committers.data.ts

```diff
@@ -0,0 +1,48 @@
+import { Octokit } from "octokit";
+import { DefaultTheme } from "vitepress";
+
+const GITHUB_TOKEN = process.env.GITHUB_TOKEN || "";
+const octokit = new Octokit({ auth: GITHUB_TOKEN });
+
+export default {
+  async load() {
+    const contributors = await (async () => {
+      try {
+        return await octokit.paginate(octokit.rest.repos.listContributors, {
+          owner: "FabricMC",
+          repo: "fabric-docs",
+        });
+      } catch (e) {
+        // Allows build without internet
+        return [];
```

Comment:
This isnt good as it will silently fail if there is an error making this request. It should bail out before any request is made if the github token isnt set.